lighthouse-core/gather/gatherers/content-width.js

```diff
@@ -24,9 +24,11 @@ const Gatherer = require('./gatherer');
 function getContentWidth() {
   // window.innerWidth to get the scrollable size of the window (irrespective of zoom)
   // window.outerWidth to get the size of the visible area
+  // window.devicePixelRatio to get ratio of logical pixels to physical pixels
   return Promise.resolve({
     scrollWidth: window.innerWidth,
-    viewportWidth: window.outerWidth
+    viewportWidth: window.outerWidth,
+    devicePixelRatio: window.devicePixelRatio,
```

Comment:
eh, I actually opened it expecting to find it there since `content-width` for me says information about the different widths (CSS pixel width, inner width, actual width, etc) but I can see it being unintuitive. Ideas on better names?